integration_tests/testsuite/test_logging_custom.py

```diff
@@ -48,22 +47,28 @@ def runTest(self):
                          'token is {1}, '
                          'level is {2}'.format(log_name, token, level))
 
-            self.assertTrue(self._read_log(client, log_name, token, level),
-                            'Log entry not found for posted token!')
+            project_id = test_util.project_id()
+
+            FILTER = 'resource.type = "global" AND logName = ' \
```

Comment:
well, we could also pass it in to the request for the _custom_ logging - the Logging API at least enables the resource to be changed on the client side - I'm explicitly setting it as well. The speed difference is significant as well. But if the resource is not set (like with Java Util Logging or for the standrad test) and default discovery is used on the app side (like [in Java](https://github.com/GoogleCloudPlatform/google-cloud-java/blob/43203a7712075d31424202c0115b0a036920e316/google-cloud-logging/src/main/java/com/google/cloud/logging/MonitoredResourceUtil.java#L104)), I agree, it would be very hard to replicate that logic in the driver, externally.
I'll take it out!